recipes/muscle/build.sh

```diff
@@ -1,9 +1,18 @@
 #!/bin/bash
+set -e
 
-cd src/
-sed -i.bak -e 's/ -static//' Makefile
+mkdir -p ${PREFIX}/bin
+cd src || exit 1
+echo "0" > gitver.txt
 
-make CXX=$CXX
+cp ${RECIPE_DIR}/vcxproj_make.py .
+chmod +x vcxproj_make.py
+./vcxproj_make.py --openmp --cppcompiler ${CXX}
 
-mkdir -p "$PREFIX"/bin
-cp "$(uname)"/muscle "$PREFIX"/bin/muscle
+# Verify binary exists and is executable
+if [ ! -f ../bin/muscle ]; then
```

Comment:
```suggestion
if [ ! -x ../bin/muscle ]; then
```
This will check that it is executable